src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-skip-to-content {
```

Comment:
Let's add `user-select: none;` to avoid accidental highlighting of the text when clicking on the anchor.
![Screen Shot 2022-02-10 at 6 43 06 PM](https://user-images.githubusercontent.com/849817/153515842-f09a5f70-ad44-4777-ba18-6db34500e0e9.png)

Reply:
Fixed